packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1,7 +1,20 @@
 import type { Browser, MultiRemoteBrowser } from 'webdriverio'
 import type { Capabilities } from '@wdio/types'
+import logger from '@wdio/logger'
 
-import { BROWSER_DESCRIPTION } from './constants'
+import got from 'got'
+import { hostname, platform, type, version, arch } from 'os'
+import { promisify } from 'util'
```

Comment:
Let's have node related imports at the top.

Reply:
use `node:os` and `node:util`